package.json

```diff
@@ -97,7 +96,7 @@
     "pino-http": "^6.6.0",
     "prom-client": "^13.1.0",
     "rate-limiter-flexible": "^2.2.2",
-    "redlock": "^4.2.0",
+    "redlock": "^5.0.0-beta.2",
```

Comment:
do you think is safe to use a beta version? also please remove yarn.lock and create it again

Reply:
yes I think so: the beta 2 was published 3 months ago, and the package has 250k download weekly/1m monthly.